Model/Method/Emandate.php

```diff
@@ -105,6 +105,7 @@ public function __construct(
         Factory $configProviderFactory = null,
         MethodFactory $configProviderMethodFactory = null,
         PriceHelper $priceHelper = null,
+        \Buckaroo\Magento2\Model\SecondChanceRepository $secondChanceRepository,
```

Comment:
use class alias, keep consistency for the __construct parameters.

Reply:
Thank you, fixed